hack/check_pre_req.sh

```diff
@@ -122,7 +122,8 @@ is_installed_kubectl() {
 is_installed_make() {
     is_installed 'GNU Make' \
         'make' \
-        'make --version | head -1'
+        'make --version | head -1' \
+        'sudo apt-get install make or brew install make'
```

Comment:
The situation on a Mac is more complicated. Developers generally need the Xcode Command Line Tools, which include a version of `make`. Additionally developers can install more advanced versions of various tools. I have chosen not to, and that is working for me.